tests/foreman/api/test_reporttemplates.py

```diff
@@ -455,6 +455,82 @@ def test_positive_applied_errata(
     assert res[0]['issued']
 
 
+@pytest.mark.tier2
+@pytest.mark.rhel_ver_match('[^6]')
+@pytest.mark.no_containers
+def test_positive_applied_errata_report_with_invalid_errata(
+    function_org, function_location, function_lce, rhel_contenthost, target_sat
+):
+    """Generate an Applied Errata report after an invalid errata has been applied
+
+    :id: cf64f193-870d-4053-ae4b-28148424b2e2
+
+    :setup: A Host with some invalid applied errata.
+
+    :steps:
+
+        1. Apply invalid errata
+        2. Generate an Applied Errata report
+
+    :expectedresults: A report is generated without failures
+
+    :CaseImportance: High
+
+    :BZ: 2176368
+    """
+    activation_key = target_sat.api.ActivationKey(
+        environment=function_lce, organization=function_org
+    ).create()
+    cv = target_sat.api.ContentView(organization=function_org).create()
```

Comment:
```suggestion
```
We could probably use `function_ak_with_cv` fixture instead (and `function_promoted_cv` it uses inside).

Reply:
Had some issues with lce using that fixture, Test was failing, Lets just keep it how it is

Reply:
I have also found that fixture fails to setup as expected